cmd/ci-secret-generator/main.go

```diff
@@ -64,17 +55,17 @@ type fieldGenerator struct {
 
 func parseOptions() options {
 	var o options
-	flag.CommandLine.BoolVar(&o.dryRun, "dry-run", true, "Whether to actually create the secrets with bw command")
-	flag.CommandLine.StringVar(&o.configPath, "config", "", "Path to the config file to use for this tool.")
-	flag.CommandLine.StringVar(&o.bootstrapConfigPath, "bootstrap-config", "", "Path to the config file used for bootstrapping cluster secrets after using this tool.")
-	flag.CommandLine.BoolVar(&o.validate, "validate", true, "Validate that the items created from this tool are used in bootstrapping")
-	flag.CommandLine.BoolVar(&o.validateOnly, "validate-only", false, "If the tool should exit after the validation")
-	flag.CommandLine.StringVar(&o.outputFile, "output-file", "", "output file for dry-run mode")
-	flag.CommandLine.StringVar(&o.bwUser, "bw-user", "", "Username to access BitWarden.")
-	flag.CommandLine.StringVar(&o.bwPasswordPath, "bw-password-path", "", "Path to a password file to access BitWarden.")
-	flag.CommandLine.StringVar(&o.logLevel, "log-level", "info", fmt.Sprintf("Log level is one of %v.", logrus.AllLevels))
-	flag.CommandLine.IntVar(&o.maxConcurrency, "concurrency", 1, "Maximum number of concurrent in-flight goroutines to BitWarden.")
-	if err := flag.CommandLine.Parse(os.Args[1:]); err != nil {
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
Those two are the exact same thing https://golang.org/pkg/flag/#pkg-variables :)